packages/react/src/components/Cta/__stories__/Cta.stories.js

```diff
@@ -12,70 +12,109 @@ storiesOf('Components|CTA', module)
+          copy: copy[0],
+        };
+        break;
+      case 'card':
+        type = select('type', types, types[0]);
+        data = {
+          type: type,
+          href: urlBy[type],
+          title: copy[0],
+        };
+        break;
+      case 'feature':
+        type = select('type', types, types[0]);
+        data = {
+          heading: copy[0],
+          card: {
+            type: type,
+            href: urlBy[type],
+            title: copy[1],
+            image: {
+              defaultImage: 'https://picsum.photos/id/672/672',
+              alt: '...',
+            },
+          },
+        };
+        break;
+      case 'button':
+        type = [
+          select('button 1 type', types, types[0]),
+          select('button 2 type', types, types[0]),
+        ];
+
+        data = [
+          {
+            type: type[0],
+            href: urlBy[type[0]],
+            copy: copy[0],
+          },
+          {
+            type: type[1],
+            href: urlBy[type[1]],
+            copy: copy[1],
+          },
+        ];
+        break;
+      default:
+    }
 
     return (
-      <CTA
-        // cta={select('ctaType', cta, cta.card)}
-        //type={select('type', type, type.local)}
-        style={style}
-        cta={data.button}
-        // buttons={buttons}
-      />
+      <div className="bx--grid ">
+        <div className="bx--row">
+          <div className="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
+            <CTA style={style} cta={data} />
+          </div>
+        </div>
+        {type === 'jump' || type[0] === 'jump' || type[1] === 'jump' ? (
+          <div
+            className="bx--row"
+            style={{ marginTop: '80px', marginBottom: '80px' }}>
+            <div className="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
+              {(() => {
+                let content = [];
+                for (let i = 0; i < 10; i++) {
+                  if (i === 2) {
+                    content.push(
+                      <h4 id="example" style={{ marginBottom: '32px' }}>
+                        Example
+                      </h4>
+                    );
+                  }
+                  content.push(
+                    <p style={{ marginBottom: '32px' }}>
+                      {
+                        'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
+                      }
+                    </p>
+                  );
+                }
+                return content;
+              })()}
+            </div>
+          </div>
+        ) : null}
+      </div>
```

Comment:
This section seems pretty heavy, but it automatically generates content based allowing us to demonstrate jumping down the page with smooth scroll.